lib/punchblock/translator/asterisk/component/record.rb

```diff
@@ -80,7 +80,8 @@ def filename
           end
 
           def recording
-            Punchblock::Component::Record::Recording.new :uri => "file://#{filename}.#{@format}"
+            uri = "file://#{filename}.#{@format}".gsub('wav49', 'WAV')
```

Comment:
One more it: to be safe, the `gsub` really should only be applied to `@format` and not the entire filename